rero_ils/modules/notifications/mappings/v7/notifications/notification-v0.0.1.json

```diff
@@ -9,6 +9,9 @@
       "pid": {
         "type": "keyword"
       },
+      "status": {
+        "type": "keyword"
+      },
```

Comment:
pay attention when merging to update the ES index (and rebuild it for existing notifications ?)